postgrest.cabal

```diff
@@ -115,6 +115,8 @@ library
                     , heredoc                   >= 0.2 && < 0.3
                     , http-types                >= 0.12.2 && < 0.13
                     , insert-ordered-containers >= 0.2.2 && < 0.3
+                    , interpolatedstring-perl6  >= 1 && < 1.1
```

Comment:
This depends on template haskell. @wolfgangwalther I think you're currently removing all template haskell dependencies for enabling cross compiling right?

Reply:
Whoops, yes, that's rebase, will fix in a jiff